runtime/src/main/java/org/corfudb/runtime/exceptions/TrimmedException.java

```diff
@@ -30,7 +30,7 @@ public TrimmedException() {
     }
 
     public TrimmedException(List<Long> trimmed) {
-        super(String.format("Trimmed addresses {}", trimmed));
+        super(String.format("Trimmed addresses %s", trimmed));
```

Comment:
Is it necessary to call the super constructor ?

Since the list is being consumed directly on exception, is the message really necessary?

Reply:
We want a message to be printed relevant to the trimmed addresses, as this exception is directly caught by upper layers, otherwise, we're subject to them printing this info.